### PR TITLE
enhancement : Front end implementation for speaker count

### LINKED ISSUE #1936: Events Dashboard: Show Speakers Total, Pending, Accepted, Rejected
In the list of speakers on the events dashboards (https://eventyay.com/events/live), please show speakers in the table:
* Total
* Accepted
* Confirmed
* Pending
* Rejected

screenshot from 2019-01-21 23-52-27

### PR DESCRIPTION
**Pull request by uds5501**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Implements the front end as required in the enhancement request 

#### Changes proposed in this pull request:

- Creating a new component similar to cell-session-dashboard
- Adding tests for
- Adding attributes in event-statistics-general following hook changes

image

Fixes #1936

### CODE REVIEW

**Comment by uds5501** [comment]
Pushing corresponding server PR soon

File: app/controllers/events/list.js [hunk 1]
```diff
@@ -28,7 +28,8 @@ export default Controller.extend({
       disableFiltering : true
     },
     {
-      propertyName     : 'speakers.length',
+      propertyName     : 'speakersByState',
```
Comment (CosmicCoder96) [inline app/controllers/events/list.js]: where is speakersByState defined?

Comment (uds5501) [reply app/controllers/events/list.js]: Nowhere, I didn't know what did the `propertyName` signify, so I was just trying out names during the testing and has been left here. should I keep it as `speakers.length` as it was before? @CosmicCoder96 .

Comment (CosmicCoder96) [reply app/controllers/events/list.js]: yes, please. Thank you.

Comment (uds5501) [reply app/controllers/events/list.js]: @CosmicCoder96 Done. Please review

**Pull request merged**